# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../xosd/gcc-4.0.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

Fix compilation with gcc-4.0.

  - Rene Rebe <rene@exactcode.de>

--- xosd-2.2.2/src/libxosd/xosd.c.vanilla	2005-08-30 19:02:56.000000000 +0200
+++ xosd-2.2.2/src/libxosd/xosd.c	2005-08-30 19:03:04.000000000 +0200
@@ -54,7 +54,7 @@
 #define SLIDER_WIDTH 0.8
 
 	const char* osd_default_font="-misc-fixed-medium-r-semicondensed--*-*-*-*-c-*-*-*";
-static const char* osd_default_colour="green";
+ const char* osd_default_colour="green";
 
 //const char* osd_default_font="adobe-helvetica-bold-r-*-*-10-*";
 //const char* osd_default_font="-adobe-helvetica-bold-r-*-*-10-*";
